src/test/java/hudson/tasks/junit/pipeline/JUnitResultsStepTest.java

```diff
@@ -496,18 +496,24 @@ private static Predicate<FlowNode> stageForName(final String name) {
     }
 
     public static void assertBranchResults(WorkflowRun run, int suiteCount, int testCount, int failCount, String branchName, String stageName,
-                                           String innerStageName) {
+                                           String innerStageName, boolean keepTestNames) {
```

Comment:
Doesn't this break backward compatibility? This method is used in https://github.com/jenkinsci/pipeline-maven-plugin/blob/d4251229c91710028ab297d8a4d70a7f71385500/pipeline-maven/src/test/java/org/jenkinsci/plugins/pipeline/maven/WithMavenStepOnMasterTest.java#L1282-L1283

Reply:
Fixed.